ci/licenses_golden/licenses_flutter

```diff
@@ -11,6 +11,8 @@ LIBRARY: txt
 ORIGIN: ../../../flutter/LICENSE
 TYPE: LicenseType.bsd
 FILE: ../../../flutter/.clang-tidy
+FILE: ../../../flutter/.gradle/6.5/fileChanges/last-build.bin
```

Comment:
why do we need these `.gradle/**` files? I think they can be reverted in case you didn't intend to commit them.

Reply:
@blasten The `format_and_dart_test` failed and said that `licenses_flutter` needed that file added on line 14. 

```
Collecting licenses for fuchsia
    Skipping this component - no change in signature
Collecting licenses for gpu
    Skipping this component - no change in signature
Collecting licenses for third_party
    Skipping this component - no change in signature
Collecting licenses for skia
    Skipping this component - no change in signature
============================= ERROR =============================
License script got different results than expected for out/license_script_output/licenses_flutter.
Please rerun the licenses script locally to verify that it is
correctly catching any new licenses for anything you may have
changed, and then update this file:
  flutter/sky/packages/sky_engine/LICENSE
For more information, see the script in:
  https://github.com/flutter/engine/tree/master/tools/licenses
```